tests/core/course_module_created/statements.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+        "actor": {
+            "name": "test_fullname",
+            "account": {
+                "homePage": "http:\/\/www.example.org",
+                "name": "1"
+            }
+        },
+        "verb": {
+            "id": "http:\/\/adlnet.gov\/expapi\/verbs\/completed",
+            "display": {
+                "en": "created"
+            }
+        },
+        "object": {
+            "id": "http:\/\/www.example.org\/mod\/book\/view.php?id=1",
+            "definition": {
+                "type": "http:\/\/id.tincanapi.com\/activitytype\/lms\/module",
+                "name": {
+                    "en": "lecture 1"
+                }
+            }
+        },
+        "timestamp": "2015-06-10T15:31:41+01:00",
+        "context": {
+            "platform": "Moodle",
+            "language": "en",
+            "extensions": {
+                "http:\/\/lrs.learninglocker.net\/define\/extensions\/info": {
+                    "http:\/\/moodle.org": "1.0.0",
+                    "https:\/\/github.com\/xAPI-vle\/moodle-logstore_xapi": "0.0.0-development",
+                    "event_name": "\\mod_resource\\event\\course_module_created",
+                    "event_function": "\\src\\transformer\\events\\core\\course_module_course_module_created"
+                }
```

Comment:
`course_module_course_module_created` should be `course_module_created`

Reply:
Made that change but still get the same error